app/controllers/customer_orders_controller.rb

```diff
@@ -0,0 +1,36 @@
+class CustomerOrdersController < ApplicationController
+  before_action :authenticate_user!, only:[:new, :edit, :index]
+
+  def index
+    @customer_orders = CustomerOrder.where(user_id: current_user.id)
+    @prices = total(@customer_orders)
+  end
+
+  def total(customer_orders)
+    @total = 0
+
+    @customer_orders.each do |order|
+      @total += order.product.price
+    end
+
+    @total
+  end
+
+  def show
+    customer_order
+  end
+
+  def destroy
```

Comment:
if a customer, they dont have any more action like this. they just cancel our orders. please make a cancel method to do this.